File: app/src/main/java/com/example/anton/sprintcalendar/Format.java
package com.example.anton.sprintcalendar;

import org.joda.time.LocalDate;
import org.joda.time.format.DateTimeFormat;
import org.joda.time.format.DateTimeFormatter;

public class Format {

    private static final DateTimeFormatter DATE_FORMATTER = DateTimeFormat.forPattern("dd.MM.yyyy");

    public static String date(LocalDate date) {
        return date != null ? DATE_FORMATTER.print(date) : "<null>";
    }
}

File: app/src/main/kotlin/net/sourcefusion/agiletools/sprintcalendar/Format.kt
package net.sourcefusion.agiletools.sprintcalendar

import org.joda.time.LocalDate
import org.joda.time.format.DateTimeFormat

object Format {
    private val DATE_FORMATTER = DateTimeFormat.forPattern("dd.MM.yyyy")
    @JvmStatic fun date(date: LocalDate?): String {
        return if (date != null) DATE_FORMATTER.print(date) else "<null>"
    }
}
